paddle/framework/channel_test.cc

```diff
@@ -82,6 +82,33 @@ TEST(Channel, ConcurrentSendNonConcurrentReceiveWithSufficientBufferSize) {
   delete ch;
 }
 
+void ReceiveEmptyChannelTest(Channel* ch) {
```

Comment:
The  prefix Test is a convention of unit test names.  This function is not a unit test; a better name could be `ReceiveFromEmptyChannelShouldBlock`.